gobblin-modules/gobblin-kafka-09/build.gradle

```diff
@@ -87,6 +87,9 @@ artifacts {
 }
 
 test {
+  // this module's test cases hang often when running in parallel.
+  // making every test class execute in a forked test process till tests are fixed
+  forkEvery = 1
```

Comment:
Q: Is this the only change targeting for the travis issue ?

Reply:
yes.